glusterd2/volume/struct.go

```diff
@@ -163,9 +163,9 @@ func NewBrickEntries(bricks []api.BrickReq, volName, volfileID string, volID uui
 		}
 
 		switch b.Type {
-		case "arbiter":
+		case "arbiter", "Arbiter":
```

Comment:
still holds good

Reply:
Handles more cases, not sure what values can be passed.It might not be useful now but this is not affecting performance.

Reply:
this will  allow the user to send different types of bricks, we should not consider `Arbiter` and `arbiter` as  same,
if this is not required please remove it.

Reply:
https://github.com/gluster/glusterd2/pull/1271#discussion_r225127063